lib/osm_community_index.rb

```diff
@@ -0,0 +1,26 @@
+module OsmCommunityIndex
+  def self.add_to_i18n
+    communities = Community.all
+    files = Dir.glob(Rails.root.join("node_modules/osm-community-index/i18n/*"))
+    files.each do |file|
+      locale = File.basename(file, ".yaml")
+      community_locale_yaml = YAML.safe_load(File.read(file))[locale]
+      # rails wants en-GB but osm-community-index has en_GB
+      locale_rails = locale.tr("_", "-")
+      data = {}
+
+      communities.each do |community|
```

Comment:
I'd use `each_with_object` here rather than initialising `data` before and then using it in the loop.

Reply:
That's much nicer - done.